routers/web/repo/pull.go

```diff
@@ -652,6 +652,24 @@ func PrepareViewPullInfo(ctx *context.Context, issue *issues_model.Issue) *git.C
 	}
 
 	if pb != nil && pb.EnableStatusCheck {
+
+		var missingRequiredChecks []string
+		for _, requiredContext := range pb.StatusCheckContexts {
+			contextFound := false
+			for _, presentStatus := range commitStatuses {
+				if presentStatus.Context == requiredContext {
+					contextFound = true
+					break
+				}
+			}
+
+			if !contextFound {
+				missingRequiredChecks = append(missingRequiredChecks, requiredContext)
+			}
+		}
+		ctx.Data["MissingRequiredChecks"] = missingRequiredChecks
+
+		ctx.Data["RequiredStatusChecks"] = pb.StatusCheckContexts
```

Comment:
```suggestion
```

By the way, is this `RequiredStatusChecks` used anywhere? If no, it could be removed.

Reply:
I grepped through the code and this is the only place where ` RequiredStatusChecks` is present, so I removed it :+1: